docs/architecture-decisions/0003-implement-as-a-rest-api.md

```diff
@@ -25,6 +25,24 @@ of a subject matter expert in the libraries.
 This pattern would also replace the 'fact' feature built into the TIMDEX UI. In other words, TACOS would be responsible
 for matching DOIs, ISBNS, ISSNs, and other identifiers that could resolve known-item searches.
 
+## GraphQL vs REST
+
+### REST
+
+For a single endpoint API like we are imagining, the simplicity of REST is a nice benefit. We are imagining a minimal parameters and a single structured JSON data response and REST is very clean to use for this purpose.
+
+If we choose REST, we should assume we'll need to include a breaking version at some point in the future and start our routes versioned (i.e. `/api/v1?searchterm?=` rather than just `/?searchterm=`).
+
+### GraphQL
+
+GraphQL requires a bit more care to implement from a consumption perspective as it is likely we'd break each portion of the structured JSON data into an optional returned value so consumers would need to explicitly ask for the data they plan to use. This can also be a benefit as we'd only send consumers the data they need, so while there is friction it is fairly small once consumers embrace how GraphQL works.
+
+GraphQL would allow us to have an interactive dashboard, GraphiQL, similar to what we have for TIMDEX to allow for an easy way to see responses from the API in a simple, self documenting web UI.
+
+While we don't anticipate a need for this so early on, GraqphQL's deprecation instead of versioning could come in handy if this API is long lived. We were able to leverage this well in TIMDEX.
```

Comment:
Typo: `GraphQL's deprecation`

Reply:
I read this 10 times before I saw what the issue was lol. Thanks!

Reply:
I believe the phrasing has not changed yet. I'm pushing up a commit to fix the noted typos now.